app/javascript/components/OfficeFilter/index.js

```diff
@@ -1,22 +1,53 @@
 import React from 'react'
 
-import MenuItem from 'material-ui/MenuItem'
+import * as R from 'ramda'
+import { Dropdown, Field, Item, Menu, Select } from '@zendeskgarden/react-dropdowns'
 import { withNamespaces } from 'react-i18next'
+import { Skeleton } from '@zendeskgarden/react-loaders'
 
-import Filter from 'components/Filter'
-import { styles } from 'components/Filter'
+const Loader = withNamespaces()(({ t }) => (
+  <Dropdown>
+    <Field>
+      <Select selectedItem="">
+        <strong>{t('volunteer_portal.dashboard.layouteventstab.office')}</strong> <Skeleton width="4rem" />
+      </Select>
+    </Field>
+    <Menu>
+      {[1, 2, 3, 4].map(i => (
```

Comment:
What's your opinion on `Array(4).fill().map((_, i) => ...` instead? I think you used it that way in `app/javascript/pages/Dashboard/Leaderboard/index.js`

Reply:
I didn't here because 4 is such a small number and I save on an iteration. But I am not really fused. If it's for consistency then I can change it.